Tools/autotest/sim_vehicle.py

```diff
@@ -493,7 +501,7 @@ def options_for_frame(frame, vehicle, opts):
     if frame in _options_for_frame:
         ret = _options_for_frame[frame]
     else:
-        for p in ["octa", "tri", "y6", "firefly", "heli", "last_letter", "jsbsim", "quadplane", "plane-elevon", "plane-vtail", "plane"]:
+        for p in ["octa", "tri", "y6", "firefly", "heli", "gazebo", "last_letter", "jsbsim", "quadplane", "plane-elevon", "plane-vtail", "plane"]:
```

Comment:
I didn't try yet, but it shouldn't be Gazebo ? or change other Gazebo to gazebo (this should be more consistant with other default name)